### PR TITLE
RememberMe

### PR DESCRIPTION
**Pull request by noobsi**
remember

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -14,6 +15,20 @@ class User < ApplicationRecord
   before_save :email_downcase
   has_secure_password
 
+  def remember
+    remember_token = User.new_token
+    update_attribute :remember_digest, User.digest(remember_token)
```
Comment (ducbet) [inline app/models/user.rb]: dùng update_attributes nhé

File: app/models/user.rb [hunk 2]
```diff
@@ -14,6 +15,20 @@ class User < ApplicationRecord
...
+  end
+
+  def authenticated? remember_token
+    return false if remember_digest.nil?
+    BCrypt::Password.new(remember_digest).is_password? remember_token
+  end
+
+  def forget
+    update_attribute :remember_digest, nil
```
Comment (ducbet) [inline app/models/user.rb]: update_attributes

File: app/models/user.rb [hunk 3]
```diff
@@ -14,6 +15,20 @@ class User < ApplicationRecord
   before_save :email_downcase
   has_secure_password
 
+  def remember
+    remember_token = User.new_token
```
Comment (thaotonto) [inline app/models/user.rb]: @remember_token

**Comment by thaotonto** [comment]
ok

**Comment by ducbet** [comment]
ok

**Pull request merged**